src/masternode-budget.cpp

```diff
@@ -1334,6 +1334,25 @@ CBudgetProposal::CBudgetProposal(const CBudgetProposal& other)
     strInvalid = "";
 }
 
+// initialize from network broadcast message
+bool CBudgetProposal::LoadBroadcast(CDataStream& broadcast)
+{
+    *this = CBudgetProposal();
```

Comment:
This doesn't look really well, wouldn't be better to make a constructor accepting a `CDataStream` or make this method static and return an Optional?

Reply:
Initially wrote this as a constructor. 
Later I've changed it to a boolean function, as we might want to check if the stream can be serialized without making `ProcessMessage` throw an exception. Can rename `Load` to `Parse` if preferred.